lib/ts/recipe/session/sessionAuth.tsx

```diff
@@ -28,6 +28,7 @@ import { SessionClaimValidator } from "supertokens-website";
 
 export type SessionAuthProps = {
     requireAuth?: boolean;
+    doRedirection?: boolean;
```

Comment:
Add a comment for the user like -> "To know the behaviour of requireAuth, please visit ..."

Reply:
![image](https://user-images.githubusercontent.com/1129990/186131539-720795eb-1be5-46ed-826b-6aab5284b4bc.png)